sanic/sanic.py

```diff
@@ -296,18 +295,11 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         :param sock: Socket for the server to accept connections from
         :param workers: Number of processes
                         received before it is respected
-        :param loop: asyncio compatible event loop
         :param protocol: Subclass of asyncio protocol class
         :return: Nothing
         """
         self.error_handler.debug = debug
         self.debug = debug
-        if loop is not None:
```

Comment:
aaah, I see 👍  that is an even better idea, since I want to see the loop removed as soon as possible -- but wait that also doesn't make sense: if it's already been deprecated, why give a deprecation warning? 🤔